sys/src/cmd/newfs/build.json

```diff
@@ -0,0 +1,14 @@
+{
+	"newfs": {
+		"CFlags": [],
```

Comment:
sure.. I  just based  this on `sys/src/cmd/acme/build.json`, there's also an empty one in there.